pkg/openid4vp/openid4vp.go

```diff
@@ -148,8 +148,31 @@ func (o *Interaction) VerifierDisplayData() (*VerifierDisplayData, error) {
 	}, nil
 }
 
+type presentOpts struct {
+	ignoreConstraints bool
+}
+
 // PresentCredential presents credentials to redirect uri from request object.
 func (o *Interaction) PresentCredential(credentials []*verifiable.Credential) error {
+	return o.presentCredentials(
+		credentials,
+		&presentOpts{},
+	)
+}
+
+// PresentCredentialUnsafe presents a single credential to redirect uri from request object.
+// This skips presentation definition constraint validation.
+func (o *Interaction) PresentCredentialUnsafe(credentials []*verifiable.Credential) error {
```

Comment:
Switch it from an array of credentials to a single credential and then you can remove the multiple credential check

Reply:
Done